akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterShardingSettings.scala

```diff
@@ -22,6 +22,24 @@ object ClusterShardingSettings {
   val StateStoreModePersistence = "persistence"
   val StateStoreModeDData = "ddata"
 
+  /**
+   * INTERNAL API
+   */
+  @InternalApi
+  private[akka] val RememberEntitiesStoreCustom = "custom"
```

Comment:
is this needed?

Reply:
It's used in settings to validate config and ClusterSharding when initializing, so alternative is two hardcoded strings in those places. No strong opinion.

Reply:
I didn't find any references to it, but I could have missed something. Maybe add some comment about that it's for testing.